_docs/releasing_version.md

```diff
@@ -47,19 +61,18 @@ the changelog generator expects the token in the environment variable `CHANGELOG
 CHANGELOG_GITHUB_TOKEN='mytoken' bundle exec rake changelog
 ```
 
-The changelog generator checks for certain labels on closed issues and PRs since
-the last release and groups them together. If the changes were neither
+The changelog generator checks for certain labels on closed issues and PRs
+since the last release and groups them together. If the changes were neither
 backwards-incompatible nor only bug fixes, make a minor release. Check the
 generated diff for the CHANGELOG.md. If your chosen release version doesn't
-match the generated changelog, update metadata.json and run the changelog task again.
+match the generated changelog, update metadata.json and run the changelog task
+again. Ensure the pull requests in the diff have the appropriate GitHub labels.
+If not, update the label for the PR and re-run the changelog task.
 
 Get community feedback on the release pr, label it with skip-changelog, get it merged.
 
-Checkout an updated copy of master
 
-```bash
-git checkout master; git fetch origin; git pull origin master
-```
+If you do the merge, you should do the following release rake task.
```

Comment:
I think we should recommend against merge commits for release PRs. My main reason is that you end up with tagging the merge commit this way, which I think is ugly. So either rebase (preferred) or squash merging.

Reply:
On the other hand, when doing rebase/squash from the GitHub GUI, that means commits or rewritten and signed commits cannot be verified anymore.  "Merging" the branch from git (`git merge --ff-only release`) and pushing should close the related PR I think, not as convenient as clicking "Merge" but might be a workaround?

Reply:
Right, I didn't want to suggest we always merge via command line, but I use https://github.com/theforeman/foreman-installer-modulesync/blob/master/bin/release-merge. Note the actual releasing of the module is a bit different than what Vox Pupuli does, but https://github.com/theforeman/foreman-installer-modulesync/blob/master/bin/release-module is there for reference. And https://github.com/theforeman/foreman-installer-modulesync/blob/master/bin/release-pr is how I create the release PRs themselves. There are some git aliases in there that you may not have, but I think it should be mostly clear.

Note there are some things like passing in branches, which is there to maintain stable branches which Vox doesn't do.